### PR TITLE
Add readmes

### PR DESCRIPTION
**Pull request by dberthault**

### CODE REVIEW

**Comment by dgupta-hqs** [comment]
I think you might have forgotten to run `cargo licensepull` to update the `DEPENDENCIES`

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 This changelog track changes to the qoqo-qir project starting at version v0.1.0
 
+## 0.1.1
+
+* Add READMEs
```
Comment (kbarkhqs) [inline CHANGELOG.md:7]:
```suggestion
* Added READMEs
```

File: qoqo_qir/README.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+[![GitHub Workflow Status](https://github.com/HQSquantumsimulations/qoqo_qir/workflows/ci_tests/badge.svg)](https://github.com/HQSquantumsimulations/qoqo_qir/actions)
+[![PyPI](https://img.shields.io/pypi/v/qoqo_qir)](https://pypi.org/project/qoqo_qir/)
+[![PyPI - Format](https://img.shields.io/pypi/format/qoqo_qir)](https://pypi.org/project/qoqo_qir/)
+![Crates.io](https://img.shields.io/crates/l/qoqo-qir)
+
+QIR interface for the qoqo quantum toolkit by [HQS Quantum Simulations](https://quantumsimulations.de).
+
+qoqo-qir provides the QirBackend class that allows users translate a qoqo circuit into a QIR file.
+Not all qoqo operations have a corresponding QIR expression.  
+Circuits containing operations without a corresponding expression can not be translated.
```
Comment (kbarkhqs) [inline qoqo_qir/README.md:13]:
```suggestion
Circuits containing operations without a corresponding expression cannot be translated.
```

File: roqoqo-qir/README.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+[![Crates.io](https://img.shields.io/crates/v/roqoqo-qir)](https://crates.io/crates/roqoqo-qir)
+[![GitHub Workflow Status](https://github.com/HQSquantumsimulations/qoqo_qir/workflows/ci_tests/badge.svg)](https://github.com/HQSquantumsimulations/qoqo_qir/actions)
+[![docs.rs](https://img.shields.io/docsrs/roqoqo-qir)](https://docs.rs/roqoqo-qir/)
+![Crates.io](https://img.shields.io/crates/l/roqoqo-qir)
+
+QIR interface for the roqoqo quantum toolkit by [HQS Quantum Simulations](https://quantumsimulations.de).
+
+roqoqo-qir provides the QirBackend class that allows users translate a roqoqo circuit into a QIR file.
+Not all roqoqo operations have a corresponding QIR expression.  
+Circuits containing operations without a corresponding expression can not be translated.
```
Comment (kbarkhqs) [inline roqoqo-qir/README.md:13]:
```suggestion
Circuits containing operations without a corresponding expression cannot be translated.
```

**Review by kbarkhqs — APPROVED** [review]

**Pull request merged**